models/models.py

```diff
@@ -0,0 +1,40 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import UserMixin
+from enums.cameraEnums import CameraMode, CameraStatus
+from enums.eventEnums import EventType
+
+db = SQLAlchemy()
+
+class Users(db.Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(50), unique=True, nullable=False)
+    password = db.Column(db.String(80), nullable=False)
+    events = db.relationship('Event', backref='users', lazy=True)
+    cameras = db.relationship('Camera', backref='users', lazy=True)
+
+    def create(self):
+        db.session.add(self)
+        db.session.commit()
```

Comment:
Extracted all the DB models to their own package. The `create` method is used to communicate with the DB and insert records.